std/system.d

```diff
@@ -82,5 +82,76 @@ immutable
     /// The endianness that the program was compiled for.
     version (LittleEndian) Endian endian = Endian.littleEndian;
     else                  Endian endian = Endian.bigEndian;
+    /++
+        CPU architecture.
+
+        Note:
+            This is intended for cases where you need a value representing the
+            CPU architecture at runtime. If you're doing something which should
+            compile differently depending on CPU architecture, then please use
+            `version (X86_64)`, `version (ARM)`, etc.
+
+        See_Also:
+            $(DDSUBLINK spec/version,PredefinedVersions, Predefined Versions)
+      +/
+    enum CPU
```

Comment:
CPU is quite vague. LLVM for example uses cpu in command line flags to denote a specific CPU model name e.g. "skylake".

Ideally it'd be called something like `BaseInstructionSet` or something like that (it can't just be instruction set because the instruction set isn't solely defined by (say) riscv64)

Reply:
Perhaps ISA would be more accurate, then.

Reply:
We should have a TargetTriple tho

Reply:
Baby steps first

Reply:
Renamed to ISA/instructionSetArchitecture. I was considering InstructionSetArchitecture/instructionSetArchitecture but that seems unnecessarily verbose, since ISA is a common acronym